requirements.txt

```diff
@@ -1,33 +1,5 @@
-alabaster==0.7.11
-asn1crypto==0.24.0
-Beaker==1.10.0
-certifi==2018.8.13
-cffi==1.11.5
-chardet==3.0.4
-click==6.7
-cryptography==2.3.1
 csh-ldap==2.4.0
-dnspython==1.15.0
-Flask==1.0.2
-Flask-pyoidc==1.3.0
-future==0.16.0
-gunicorn==19.9.0
-idna==2.7
-itsdangerous==0.24
-Jinja2==2.10
-Mako==1.0.7
-MarkupSafe==1.0
-oic==0.11.0.1
-pyasn1==0.4.4
-pyasn1-modules==0.2.2
-pycparser==2.18
-pycryptodomex==3.6.5
-pyjwkest==1.4.0
-pyldap==3.0.0.post1
-pyOpenSSL==18.0.0
-python-ldap==3.4.0
-requests==2.19.1
-six==1.11.0
-srvlookup==2.0.0
-urllib3==1.23
-Werkzeug==0.14.1
+Flask>=2.0
+Flask-pyoidc
```

Comment:
Can you specify the specify pyoidc version that you're using. Same thing about builds being more reproducible.